src/core/unit.c

```diff
@@ -2385,7 +2385,7 @@ static int unit_log_resources(Unit *u) {
 
                 assert(io_fields[k].journal_field);
 
-                (void) unit_get_io_accounting(u, k, k > 0, &value);
+                (void) unit_get_io_accounting(u, k, true, &value);
```

Comment:
```suggestion
                (void) unit_get_io_accounting(u, k, /* allow_cache = */ true, &value);
```

Also this is not really obvious, hence worth a comment.

Reply:
Yup, forgot that. I'll update this as soon as I get back home :)